src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PeopleTable } from '../components/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState('');
+  const getData = async () => {
+    setIsLoading(true);
+    try {
+      const data = await getPeople();
+
+      setPeople(data);
+    } catch {
+      setError('Something went wrong');
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  useEffect(() => {
+    getData();
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {/* eslint-disable */}
+          {isLoading ? (
```

Comment:
I know we have a lot of conditions here, but still, personally, it's hard to read this conditional rendering.
Maybe you could've used an additional function here?